refinery/analysis_manager/tasks.py

```diff
@@ -86,6 +87,17 @@ def get_taskset_result(task_group_id):
     return TaskSetResult.restore(task_group_id)
 
 
+def _get_tool(analysis_uuid):
+    try:
+        return tool_manager.models.Tool.objects.get(
+            analysis__uuid=analysis_uuid
+        )
+    except (tool_manager.models.Tool.DoesNotExist,
+            tool_manager.models.Tool.MultipleObjectsReturned) as e:
```

Comment:
Are both errors of the same class? I'm wondering if `DoesNotExist` is actually ok for some situations, but `MultipleObjectsReturned` indicates a deeper problem with the database?